Tests/MapboxMapsTests/MapView/OptionsIntegrationTests.swift

```diff
@@ -25,7 +23,7 @@ internal class OptionsIntegrationTest: MapViewIntegrationTestCase {
         XCTAssertEqual(mapView.gestureManager.gestureOptions, newOptions.gestures)
         XCTAssertEqual(mapView.cameraManager.mapCameraOptions, newOptions.camera)
         XCTAssertEqual(mapView.locationManager.locationOptions, newOptions.location)
-        XCTAssertFalse(
+        XCTAssertTrue(
```

Comment:
Why did the behavior change?

Reply:
Previously you had to set the `showsScale` and `showsCompass` for these to exist. Since removing those variables, now they ornaments are created, and will be based on the visibility property.